pymess/backend/emails/__init__.py

```diff
@@ -69,6 +69,22 @@ def create_message(self, sender, sender_name, recipient, subject, content, relat
         except PersistenceException as ex:
             raise self.EmailSendingError(force_text(ex))
 
+    def is_turned_on_batch_sending(self):
+        return settings.EMAIL_BATCH_SENDING
+
+    def get_waiting_messages(self):
+        return self.model.objects.filter(
```

Comment:
This filter is here several times, the only input parameters are 2 settings values and state, maybe it could be generalized into a method in base class?